src/application/service/AddressLookUpService.ts

```diff
@@ -149,13 +149,18 @@ class AddressLookUpService {
     country: string,
     uiErrors: UIErrors
   ): boolean {
+
     const isValid =
-      (jurisdiction === Jurisdiction.SCOTLAND && country === "scotland") ||
-      (jurisdiction === Jurisdiction.NORTHERN_IRELAND && country === "northern-ireland") ||
-      (jurisdiction === Jurisdiction.ENGLAND_AND_WALES && (country === "england" || country === "wales"));
+      (jurisdiction === Jurisdiction.SCOTLAND && country === "GB-SCT") ||
```

Comment:
What happens if jurisdiction is "Scotland" but country is "border"? That should be valid.

Reply:
it looks like this check is only performed on the manual address entry page, so you should only be able to select the 4 countries from the drop down